### ISSUE TITLE
api domain resolution returns incorrect value

### ISSUE THREAD

**Issue by foohonpie** [pbbg/pbbg.com#23]
Coming from dev.pbbg.com:

<img width="567" alt="Screen Shot 2020-10-18 at 9 46 09 AM" src="https://user-images.githubusercontent.com/674157/96374306-cff06600-1126-11eb-96f4-dc26d7e59229.png">

Should we set an `API_DOMAIN` in the config, vs determining in the code?

**Comment by bpkennedy** [comment]
Oh, this is an error in the axios setup right [here](https://github.com/pbbg/pbbg.com/blob/17dc2265fbcfc244ea2e1aedaff748b46657bede/src/boot/axios.js#L7):
```
const isDevEnvironment = window.location.origin.includes('dev-api.pbbg.com')
```

Should be:
```
const isDevEnvironment = window.location.origin.includes('dev.pbbg.com')
```

Hmm, the environment variable could make it more straightforward. The client wouldn't need to observe the host or make any choices - just access the variable and make it whatever that is. It does mean the developer has to ensure they have their .env again with the API_DOMAIN but that's no big deal really.  Yea I like that.

If for security reasons, it's not like environment variables will solve this for us - as developers can just hardcode the prod url in their environment variables.

**Comment by bpkennedy** [comment]
This is fixed via #22
